src/core/components/model-example.jsx

```diff
@@ -61,39 +62,58 @@ export default class ModelExample extends React.Component {
 
     let isOAS3 = specSelectors.isOAS3()
 
-    return <div className="model-example">
-      <ul className="tab">
-        <li className={ "tabitem" + ( this.state.activeTab === "example" ? " active" : "") }>
-          <a className="tablinks" data-name="example" onClick={ this.activeTab }>{isExecute ? "Edit Value" : "Example Value"}</a>
-        </li>
-        { schema ? <li className={ "tabitem" + ( this.state.activeTab === "model" ? " active" : "") }>
-          <a className={ "tablinks" + ( isExecute ? " inactive" : "" )} data-name="model" onClick={ this.activeTab }>
-            {isOAS3 ? "Schema" : "Model" }
-          </a>
-        </li> : null }
-      </ul>
-      <div>
-        {
-          this.state.activeTab === "example" ? (
-            example ? example : (
+    return (
+      <div className="model-example">
+        <ul className="tab" role="tablist">
+          <li className={cx("tabitem", { "active": this.state.activeTab === "example" })} role="presentation">
+            <button
+              aria-selected={this.state.activeTab === "example"}
+              className="tablinks"
+              data-name="example"
+              onClick={ this.activeTab }
+              role="tab"
+            >
+              {isExecute ? "Edit Value" : "Example Value"}
+            </button>
+          </li>
+          { schema && (
+            <li className={cx("tabitem", { "active": ["model", "schema"].includes(this.state.activeTab) })} role="presentation">
+              <button
+                aria-selected={["model", "schema"].includes(this.state.activeTab)}
+                className={ "tablinks" + ( isExecute ? " inactive" : "" )}
+                data-name={isOAS3 ? "schema" : "model" }
+                onClick={ this.activeTab }
+                role="tab"
+              >
+                {isOAS3 ? "Schema" : "Model" }
+              </button>
+            </li>
+          )}
+        </ul>
+        {this.state.activeTab === "example" && (
+          <div data-name="examplePanel" role="tabpanel" aria-hidden={this.state.activeTab !== "example"} tabIndex="0">
```

Comment:
Instead of using data attributes here, wouldn't be more idiomatic to use `aria-labelledby` attribute?

Reply:
I'm in favor of including `aria-labelledby`, but the data-name pattern is recommended as a best practice by cypress https://docs.cypress.io/guides/references/best-practices#Selecting-Elements. If possible, I would like to include both if we can figure out how to create unique ids for the buttons and tabpanels.

Reply:
> data-name pattern is recommended as a best practice by cypress https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Right, let's leave the data attributes in place and let's figure out how to use `aria-lalbelledby` attribute

Reply:
We already use [randombytes](https://www.npmjs.com/package/randombytes) library in SwaggerUI. Let's use it to create unique ids:

```
import randomBytes from "randombytes"

const uniqueId = randomBytes(5).toString("base64")
```

Would that be enough?

Reply:
Thank you for pointing out this utility. This is enough to add the `aria-controls `and `aria-labelledby` attributes back in.

Reply:
@char0n let me know if there are any additional change requests.